app/customer.py

```diff
@@ -0,0 +1,74 @@
+import math
+from dataclasses import dataclass
+from datetime import datetime
+from typing import List
+
+from app.car import Car
+from app.shop import Shop
+
+
+@dataclass
+class Customer:
+    name: str
+    product_cart: dict
+    location: List[int]
+    money: int
+    car: Car
+
+    def calculate_products_cost(self, shop: Shop,
+                                print_info: bool = False) -> float:
+        product_coast = sum([shop.products[key] * self.product_cart[key]
```

Comment:
The variable `product_coast` seems to be a typo. It should be `product_cost` to reflect its purpose of storing the total cost of products.